src/flask_container_scaffold/logging.py

```diff
@@ -0,0 +1,32 @@
+import logging
+
+from flask import has_request_context, request
+
+
+class FlaskRequestFormatter(logging.Formatter):
```

Comment:
I am wondering how exactly this is used in an app build on top of flask-container-scaffold.  Would you mind adding an example here in the docs (or readme if you prefer) to make it more clear how to properly use this?

Reply:
A usage example is a good idea, will update.

Reply:
Let me know if this is what you had in mind.

Reply:
That is very helpful, yes, thanks! The other part I was thinking was a bit of direction for the user on where to call this wrt to initializing their app, since we have several projects that are likely going to be coming on board as users soon.

Reply:
I updated the README. I put a full dictConfig() in there even if it would need to be tweaked for each app - happy to only show the bits relevant to the formatter like in this doc string if that makes more sense instead.

Reply:
While perhaps a hair more detail than I was thinking, I say we keep, nobody ever complained something was too clear and easy to follow!  Thanks for adding that.